### DISCUSSION TITLE
Copilot neovim does not mantain auth after reboot

### DISCUSSION THREAD

**Discussion by dan-cooke** [community/community#61941] [Copilot Conversations]
### Select Topic Area

Bug

### Body

After setting up copilot for neovim, and rebooting my machine - 

1. `:Copilot status` prints `Enabled and online`
2. `:Copilot auth` prints `Authenticated as dan-cooke`
3. `:Copilot panel` prints `Not signed in`

I get no suggestions.

## Workaround
`:Copilot auth` and sign in again but this is annoying that I have to do this everytime i reboot my machine.

## OS
 OS: Arch Linux x86_64
 Kernel: 6.4.6-arch1-1

## Versions
node 18.15.0
npm 9.5.0
copilot.vim 1.9.1
Neovim 0.9.1
dist/agent.js 1.94.0

**Comment by BaraaBasim** [comment]
Check that your authentication token is being stored properly. GitHub Copilot stores your authentication token in a file called ~/.config/nvim/coc-settings.json. Make sure that this file exists and not modified when you reboot your system.

**Comment by dan-cooke** [comment]
@BaraaBasim  thanks for your reply but I do not use Coc so that file does not exist

However I do have a directory

`~/.config/github-copilot/hosts.json` which seems to have correctly stored my oauth2  token 

Let me check if its still there after reboot

**Comment by dan-cooke** [comment]
Update:

Even after reboot my oauth token is still in there as expected, additionally I have noticed that copilot stops working as listed in the OP not only on reboot - but when my PC locks.
